tests/Exceptionless.Tests/Pipeline/EventPipelineTests.cs

```diff
@@ -843,6 +843,43 @@ public sealed class EventPipelineTests : IntegrationTestsBase {
                 Assert.Null(userDescription?.EmailAddress);
             }
         }
+        
+        [Fact]
+        public async Task WillHandleDiscardedStack() {
+            var organization = OrganizationData.GenerateSampleOrganization(_billingManager, _plans);
+            var project = ProjectData.GenerateSampleProject();
```

Comment:
Why not use our new event builder to create an event?

Reply:
No, that doesn't go through the pipeline.